src/Dependencies.re

```diff
@@ -403,6 +403,18 @@ and translateTypeExpr_ =
          [],
          [],
        )
+  | Ttuple(listExp) =>
+    let innerTypesTranslation =
+      listExp |> translateTypeExprs_(~language, ~typeVarsGen, ~typeEnv);
+    let innerTypes = innerTypesTranslation |> List.map(({typ, _}) => typ);
```

Comment:
Very inspired on the `Tconstr(path, typeParams, _)` case [below](https://github.com/cristianoc/genType/pull/71/files#diff-d11ed9535d16ed2e430d872d7a86437bR445).